doc/source/user_guide/groupby.rst

```diff
@@ -1657,7 +1661,7 @@ arbitrary function, for example:
 
    df.groupby(["Store", "Product"]).pipe(mean)
 
-where ``mean`` takes a GroupBy object and finds the mean of the Revenue and Quantity
+Where ``mean`` takes a GroupBy object and finds the mean of the Revenue and Quantity
```

Comment:
where was lower case because it was a continuation of the previous sentence. As it is now, it is an incomplete sentence.

No opposition to splitting the sentence, maybe "Here ``mean``..." instead of "Where"?

Reply:
Yes